src/docs/Components/Checkbox/previews/intro.preview.html

```diff
@@ -1,40 +1,48 @@
 <ul class="example">
   <li class="example__item">
-    <input
-      id="checkbox-01"
-      type="checkbox"
-      class="mc-checkbox"
-      name="example"
-    />
-    <span class="example__label">default</span>
+    <div class="mc-checkbox">
+      <input
+        id="checkbox-01"
+        type="checkbox"
+        class="mc-checkbox__input"
+        name="example"
+      />
+      <label for="checkbox-01" class="mc-checkbox__label">default</label>
+    </div>
   </li>
   <li class="example__item">
-    <input
-      id="checkbox-01"
-      type="checkbox"
-      class="mc-checkbox"
-      name="example"
-      checked
-    />
-    <span class="example__label">checked</span>
+    <div class="mc-checkbox">
+      <input
+        id="checkbox-01"
+        type="checkbox"
+        class="mc-checkbox__input"
+        name="example"
+        checked
+      />
+      <label for="checkbox-01" class="mc-checkbox__label">checked</label>
+    </div>
   </li>
   <li class="example__item">
-    <input
-      id="checkbox-01"
-      type="checkbox"
-      class="mc-checkbox"
-      name="example"
-      disabled
-    />
-    <span class="example__label">disabled</span>
+    <div class="mc-checkbox">
+      <input
+        id="checkbox-01"
```

Comment:
~~id="checkbox-01"~~ => id="checkbox-03"
~~for="checkbox-01"~~ => for="checkbox-03"